src/compiler/binder.ts

```diff
@@ -1398,6 +1400,15 @@ namespace ts {
             }
         }
 
+        function bindGlobalModuleExportDeclaration(node: GlobalModuleExportDeclaration) {
+            if (!file.externalModuleIndicator) {

```

Comment:
Use the appropriate helper function to check this that's been defined elsewhere.